route/route.go

```diff
@@ -606,13 +598,12 @@ func (r *Router) processEvent(ev *types.Event, reqID interface{}) error {
 	r.Metrics.Histogram(r.incomingOrPeer+"_router_event_bytes", float64(ev.GetDataSize()))
 
 	// An error here is effectively a parsing error, so return it up the stack.
-	err := ev.Data.ExtractMetadata(r.Config.GetTraceIdFieldNames(), r.Config.GetParentIdFieldNames())
-	if err != nil {
+	if err := ev.Data.ExtractMetadata(); err != nil {
```

Comment:
I thought the idea was to get rid of this call entirely - which cases do we still need it for?